packages/pierce-dom/.eslintrc.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": ["../../.eslintrc.json"],
-  "ignorePatterns": ["!**/*"],
+  "ignorePatterns": ["!**/*", "storybook-static"],
```

Comment:
why is this needed ?

Reply:
was added automatically when I run generator, will check why it was added